app/javascript/controllers/movie_controller.js

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+} from 'stimulus';
+import $ from 'jquery';
```

Comment:
I think this line could be erased. Tried it locally and everything was working fine.

Reply:
If we delete this line our javascript linter will raise an exception

`18:5   error    '$' is not defined   `